uk.co.compucorp.civicrm.hrleaveandabsences/views/admin-dashboard/components/admin-dashboard-container.html

```diff
@@ -23,3 +23,4 @@
     </ul>
   </div>
 </div>
+<div class="clear"></div>
```

Comment:
Using the HTML is better option in my opinion, this utility class is created for this purpose only. Also adding another style for what we already have is not a good option I think.